### PR TITLE
fix(card): apply component classes to card component

### PR DESCRIPTION
**Pull request by AnnaRybkina**
Extra styles have been migrated from https://github.com/fabric-ds/css/blob/next/src/components/card.css

Type|Screenshot
---|---
Single card|image
Toggles in Card|image
Dead toggle in Card|image

### CODE REVIEW

File: packages/_helpers/clickable.tsx [hunk 1]
```diff
@@ -54,8 +55,8 @@ export function Clickable({
 
   return radio || checkbox ? (
     <ToggleItem
-      labelClassName={props.labelClassName}
-      className="focus-ring focus-ring-inset absolute inset-0 h-full w-full appearance-none cursor-pointer"
+      labelClassName={classNames(props.labelClassName)}
+      className={`focus:focusable:focus focus-visible:focusable:focus-visible not-focus-visible:focusable:focus:not(:focus-visible) focusable-inset ${ccClickable.clickable}`}
```
Comment (AnnaRybkina) [inline packages/_helpers/clickable.tsx:59]: This is needed to be fixed in Drive([the task](https://trello.com/c/pjBacYeN/222-fix-focusable-issue)) but I would like to leave it here as it works

File: packages/_helpers/clickable.tsx [hunk 2]
```diff
@@ -54,8 +55,8 @@ export function Clickable({
 
   return radio || checkbox ? (
     <ToggleItem
-      labelClassName={props.labelClassName}
-      className="focus-ring focus-ring-inset absolute inset-0 h-full w-full appearance-none cursor-pointer"
+      labelClassName={classNames(props.labelClassName)}
+      className={`focus:focusable:focus focus-visible:focusable:focus-visible not-focus-visible:focusable:focus:not(:focus-visible) focusable-inset ${ccClickable.clickable}`} //TODO should be rewritten when fixed in drive, only `focusable` should be needed
```
Comment (AnnaRybkina) [inline packages/_helpers/clickable.tsx:59]: This is needed to be fixed in Drive([the task](https://trello.com/c/pjBacYeN/222-fix-focusable-issue)) but I would like to leave it here as it works

**Comment by AnnaRybkina** [comment]
[Warpify Card component](https://trello.com/c/B7sD5ZBh/226-warpify-card-component)

**Review by BalbinaK — APPROVED** [review]
LGTM! Just remember about updating `index.ts` :D

**Pull request merged**